content/tutorials/publish-subscribe.textile

```diff
@@ -79,6 +79,52 @@ blang[ruby].
 
   "See this step in Github":https://github.com/ably/tutorials/commit/0ec2c7e
 
+blang[php].
+  To start using Ably you first need to install "composer package on packagist":https://packagist.org/packages/ably/ably-php into your composer.
+
+  ```[sh]
+    composer require ably/ably-php --update-no-dev
+  ```
+
+  The client library must be instanced with the API key you copied in Step 1. API keys used with "basic authentication":/general/authentication#basic-authentication for your own servers is generally preferred, however clients running on insecure devices should always use the "token authentication scheme":/general/authentication#token-authentication instead. In this example, we use an API key for simplicity.
+
+  PHP Client library doesn't provide realtime api support, so we'll create web app that also uses Javascript to show pub/sub pattern.
+
+  Javascript will handle subscribing to messages, while PHP will be responsible for publishing messages.
+
+  First let's create @ably.php@ where we will store Ably library and Api key
+
+  ```[php]
+    <?php
+    require_once 'vendor/autoload.php';
+    $apikey = '{{ApiKey}}';
```

Comment:
Please note if you look at https://github.com/ably/tutorials/commit/5f96f5f#commitcomment-19190834, this does not mean you add that code in here in the tutorial docs, but you do in the downloadable code so as to help users who just run the code